tools/start-nodes.sh

```diff
@@ -10,34 +10,73 @@ set -e
 # We use BASE and DEV_NODES_ARRAY variables from here
 source tools/travis-common-vars.sh
 
+async_helper() {
```

Comment:
Is here any performance benifits?
Like before and after comparison of start-nodes script time.

Reply:
as for the travis, the difference is not big. around 5 seconds.

https://travis-ci.org/esl/MongooseIM/jobs/538004973#L1180
https://travis-ci.org/esl/MongooseIM/jobs/536688520#L1163